templates/CRM/Admin/Form/MessageTemplates.tpl

```diff
@@ -84,7 +84,7 @@
         <tr>
   </table>
 
-      <div id="msg_html" class="crm-accordion-wrapper crm-html_email-accordion ">
```

Comment:
Two ids with the same text were used in the form(div and textarea input), Hence `$('#msg_html').text()` returned an empty text.